cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -102,11 +106,25 @@ public void execute(Project project) {
 
             // Print warnings for conflicted jars
             if (!jBallerinaBackend.conflictedJars().isEmpty()) {
-                out.println("\twarning: Detected conflicting jar files:");
+                out.println("\twarning: Picking the latest jar since detected conflicting jar files:");
                 for (JBallerinaBackend.JarConflict conflict : jBallerinaBackend.conflictedJars()) {
-                    out.println(conflict.getWarning(project.buildOptions().listConflictedClasses()));
+                    out.println(conflict.getWarning(project.buildOptions().listConflictedClasses(), false));
                 }
             }
+
+            // Report diagnostics
+            boolean hasErrors = false;
+            for (Diagnostic d : jBallerinaBackend.jarResolveDiagnosticResult().diagnostics(false)) {
```

Comment:
Can't we make the jarResolver diagnostic a part of jBallerinaBackend diagnostic without exposing it like this?

Reply:
You are suggesting adding these jar resolver diagnostics to the same diagnosticsResult we already have in JBallerinaBackend?

Reply:
Yes, can we accumulate it to the jBallerinbackendDiagnostics similar to how we do for the diagnostics of manifest, compilation etc?

Reply:
Since we access jar resolver diagnostics after calling `emitExecutable` method inside `emit`, we can't show those diagnostics as compiler diagnostics. If we add these jar resolver diagnostics to the same `diagnosticResult` inside `JBallerinaBackend`, then we have to filter out and show only the diagnostics we didn't show at the compiler phase.